Android/MemeShareApp_Antriksh_Batra/app/src/main/java/com/example/memeshare/MainActivity.kt

```diff
@@ -0,0 +1,80 @@
+package com.example.memeshare
+
+import android.content.Intent
+import android.graphics.drawable.Drawable
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.ImageView
+import android.widget.ProgressBar
+import android.widget.Toast
+import com.android.volley.Request.Method.GET
+import com.android.volley.Response
+import com.android.volley.VolleyError
+import com.android.volley.toolbox.JsonObjectRequest
+import com.android.volley.toolbox.StringRequest
+import com.android.volley.toolbox.Volley
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+import java.lang.reflect.Method
+
+class MainActivity : AppCompatActivity() {
+    var currentImageurl:String?=null
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+
```

Comment:
Consider formatting before committing to VCS.

Reply:
ok